excel2img/excel2img.py

```diff
@@ -105,6 +105,11 @@ def export_img(fn_excel, fn_image, page=None, _range=None):
             except com_error:
                 raise Exception("Failed locating range %s"%(_range))
 
+        if autofit is not None:
+            excel.workbook.Application.Range(rng).Columns.AutoFit()
+        elif _af_range is not None:
+            excel.workbook.Application.Range(_af_range).Columns.AutoFit()
+                
```

Comment:
The meaning of autofit/_af_range function arguments as implemented is not consistent or intuitive, e.g.:

* `autofit=True, _af_range="A1:B2"` will auto-fit everything
* `autofit=False, _af_range=None` will auto-fit everything

It would be better to have a single `autofit` option which you would check for truthiness, to do the autofit. Additionally, if autofit is an instance of `str`, interpret it as a range to autofit.

(it is ok to still have separate command line options, since argparse doesn't support optional option values)